rfc/bh-rfc-1.md

```diff
@@ -0,0 +1,169 @@
+
+## 2. Motivation & Goals
+
+- **Short-Lived Branches** - Avoid confusion and drift by limiting the number of branches that require long-term maintenance and terminating most branches once their assigned release is complete.
+- **Consistent Release Process** - Standardize how we name, manage, and merge release branches.
+- **Isolated Fixes** - Ensure hotfixes do not inadvertently pick up unreleased features from the default branch.
+
+## 3. Branch Types
+
+### 3.1 The `main` Branch
+
+- The repository's default branch name is `main`
+- This branch is the primary production-ready branch that feature branches originate.
+- All changes made to a staging branch are merged to the `main` branch after the release is tested and completed.
+
+### 3.2 Staging Branches: `stage/vX.Y.Z`
+
+- Staging branches are short-lived branches used to prepare a release candidate (RC) for a specific version (X.Y.Z)
+- Staging branches are named by combining `stage/v` and the corresponding release candidate's semantic version.
+    - E.g. - For release `1.2.3` the name of the staging branch is `stage/v1.2.3`.
+- Staging branches for major and minor release versions are branched from the `main` branch.
+- Staging branches for patch release versions are branched from the last production tag.
+- Staging branches are merged and deleted after the corresponding release has been tested, reviewed, and finalized.
+- Staging branches should not squash commits prior to merging into the `main` branch.
+- Staging branch pull requests (PRs) can be merged using the "Merge" or "Rebase" strategies.
+
+### 3.3 Feature Branches
+
+- Feature branches are short-lived branches used to develop/enhance product features and fix bugs/defects.
+- Feature branches are named by combining the project management issue number and an optional, short, human-readable, and hyphenated description of the work.
+    - E.g. - For issue number `bed-1234` the feature branch name could be either `bed-1234` or `bed-1234-my-short-description`. 
+- Feature branches are branched from the `main` branch.
+- Feature branches are merged into the `main` branch and deleted after the corresponding development work has been tested, reviewed, and approved.
+- Feature branches should squash commits prior to merging into the `main` branch when possible.
+- Feature branch pull requests (PRs) can be merged using the "Merge", "Rebase", or "Squash" strategies.
+
+### 3.4 Stabilization Branches
+
+- Stabilization branches are short-lived branches used to fix bugs/defects in a given release or staged release.
+- Stabilization branches are named by combining the project management issue number and an optional, short, human-readable, and hyphenated description of the work.
+    - E.g. - For issue number `bed-1234` the feature branch name could be either `bed-1234` or `bed-1234-my-short-description`.
```

Comment:
This says feature branch instead of stabilization branch

Reply:
:facepalm: